### PR TITLE
Rework parameterization of CORL algorithms

### PR DESCRIPTION
**Pull request by Bronzila**
With this PR we aim to streamline the parameterization of the CORL agents to ensure easier usage in our experiments

### CODE REVIEW

**Comment by Bronzila** [comment]
Currently still ongoing, but feel free to leave intermediary feedback :)

File: algorithms/offline/sac_n.py [hunk 1]
```diff
@@ -217,14 +216,14 @@ def __init__(
...
                 nn.Linear(state_dim, hidden_dim),
-                activation(),
+                nn.ReLU,
                 nn.Dropout(dropout_rate),
                 *[
                     module
                     for _ in range(hidden_layers)
                     for module in (
                         nn.Linear(hidden_dim, hidden_dim),
-                        activation(),
```
Comment (gierle) [inline algorithms/offline/sac_n.py:227]: Need to instantiate ReLU

File: algorithms/offline/sac_n.py [hunk 2]
```diff
@@ -217,14 +216,14 @@ def __init__(
         self.trunk = nn.Sequential(
             *[
                 nn.Linear(state_dim, hidden_dim),
-                activation(),
```
Comment (gierle) [inline algorithms/offline/sac_n.py:220]: same here

**Review by gierle — CHANGES_REQUESTED** [review]

File: algorithms/offline/sac_n.py [hunk 3]
```diff
@@ -217,14 +216,14 @@ def __init__(
...
+                nn.ReLU,
```
Comment (gierle) [inline algorithms/offline/sac_n.py]: same here

File: algorithms/offline/sac_n.py [hunk 4]
```diff
@@ -217,14 +216,14 @@ def __init__(
...
                 nn.Dropout(dropout_rate),
                 *[
                     module
                     for _ in range(hidden_layers)
                     for module in (
                         nn.Linear(hidden_dim, hidden_dim),
-                        activation(),
+                        nn.ReLU,
```
Comment (gierle) [inline algorithms/offline/sac_n.py]: Need to instantiate ReLU

**Pull request merged**